src/vehicle.cpp

```diff
@@ -1051,7 +1063,7 @@ void vehicle::use_controls( const tripoint &pos )
 
     uimenu menu;
     menu.return_invalid = true;
-    menu.text = _( "Vehicle controls" );
+    !veh->is_furniture ? menu.text = _( "Vehicle controls" ) : menu.text = _( "Furniture controls" );
```

Comment:
Why the negation all over the place? Why put the same code into both branches of the ternary?

Btw would be nice to change `is_furniture` to a private member and expose it via a getter. Currently everybody an just change it at any time from any place.